users/templates/users/view.html

```diff
@@ -98,3 +110,12 @@ <h4>Events running</h4>
         {% endif %}
     {% endif %}
 {% endblock %}
+
+{% block bottomscripts %}
+    {{block.super}}
+    <style type="text/css">
+        .equalwidth {
+            flex: 1 1 0px !important;
+        }
+    </style>
+{% endblock %}
```

Comment:
This should really be added to `tgrsite_local.scss` rather than inline

Reply:
(If you need me to do this to avoid setting up the compile toolchain, just let me know)